{{ cookiecutter.repo_name }}/README.md

```diff
@@ -62,7 +62,7 @@
 
 ## Get Started
 
-1. Set MODULES_PATH this is the directory that contains the current project as well as dependencies like sparsity and drtools `MODULES_PATH=..`
+1. Set MODULES_PATH this is the absolute path to the directory that contains the current project as well as dependencies like sparsity and drtools `MODULES_PATH=..`
```

Comment:
Hah, your example still suggests relative path but whatever... :P

Reply:
Ah that's what you meant

Reply:
Fixed